pages/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Header = () => {
+  return (
+    <header>
+        <nav>
```

Comment:
Really great work here, @Vaishali785 . You implemented the navigation bar using all common semantic HTML tags which helps improving web accessibility. 💯